js/foundation.util.core.js

```diff
@@ -61,4 +61,18 @@ function transitionend($elem){
   }
 }
 
-export {rtl, GetYoDigits, RegExpEscape, transitionend};
+/**
+ * Call the given function once the window is loaded,
+ * or immediately if the window is already loaded.
+ * @function
+ *
+ * @param {Function} fn - function to call on window load.
+ */
+function onLoad(fn) {
+  if (document.readyState === 'complete')
+    setTimeout(() => fn(), 0);
```

Comment:
This should be `fn();` as `setTimeout` creates a new context and does not bind `_this` and nother variables in this case.